app/routes/speak.tsx

```diff
@@ -0,0 +1,56 @@
+import { MetaFunction } from "@remix-run/node";
+
+import { InternalAnchor } from "~/components/Anchor";
+import { AnchorWithArrow } from "~/components/AnchorWithArrow";
+import { Aside } from "~/components/Aside";
+import { BodyText } from "~/components/BodyText";
+import { Heading } from "~/components/Heading";
+import { Layout } from "~/components/Layout";
+import { createMeta } from "~/config";
+
+const tagline = `No prior experience or TypeScript expertise required!`;
```

Comment:
(Out of curiosity, is there a reason behind `` ` `` instead of `"` in static strings?)

Reply:
No reason, I think I'd just copied it from somewhere that was using `${...}`. 😄